src/test/java/org/folio/moduserstest/ProxyRelationshipsIT.java

```diff
@@ -0,0 +1,278 @@
+  }
+
+  @Test
+  void userMayNominateAProxy() {
+    final var userId = generateId();
+    final var proxyUserId = generateId();
+
+    final var relationship = proxiesClient.createProxyRelationship(
+      ProxyRelationship.builder()
+        .userId(userId)
+        .proxyUserId(proxyUserId)
+        .build());
+
+    assertThat(relationship.getId(), is(notNullValue()));
+    assertThat(relationship.getUserId(), is(userId));
+    assertThat(relationship.getProxyUserId(), is(proxyUserId));
+  }
+
+  @Test
+  void canGetProxyRelationship() {
+    final var userId = generateId();
+    final var proxyUserId = generateId();
+
+    final var createdRelationship = proxiesClient.createProxyRelationship(
+      ProxyRelationship.builder()
+        .userId(userId)
+        .proxyUserId(proxyUserId)
+        .build());
+
+    final var fetchedRelationship = proxiesClient.getProxyRelationship(
+      createdRelationship.getId());
+
+    assertThat(fetchedRelationship.getId(), is(createdRelationship.getId()));
+    assertThat(fetchedRelationship.getUserId(), is(userId));
+    assertThat(fetchedRelationship.getProxyUserId(), is(proxyUserId));
+  }
+
+  @Test
+  void userMayHaveMultipleProxies() {
+    final var userId = generateId();
+    final var firstProxyUserId = generateId();
+    final var secondProxyUserId = generateId();
+
+    proxiesClient.createProxyRelationship(
+      ProxyRelationship.builder()
+        .userId(userId)
+        .proxyUserId(firstProxyUserId)
+        .build());
+
+    proxiesClient.attemptToCreateProxyRelationship(
+      ProxyRelationship.builder()
+        .userId(userId)
+        .proxyUserId(secondProxyUserId)
+        .build())
+      .statusCode(is(HTTP_CREATED));
+  }
+
+  @Test
+  void sameUserMayProxyForMultipleUsers() {
+    final var proxyUserId = generateId();
+    final var firstUserId = generateId();
+    final var secondUserId = generateId();
+
+    proxiesClient.createProxyRelationship(
+      ProxyRelationship.builder()
+        .userId(firstUserId)
+        .proxyUserId(proxyUserId)
+        .build());
+
+    proxiesClient.attemptToCreateProxyRelationship(
+        ProxyRelationship.builder()
+          .userId(secondUserId)
+          .proxyUserId(proxyUserId)
+          .build())
+      .statusCode(is(HTTP_CREATED));
+  }
+
+  @Test
+  void userCannotProxyForTheSameUserMoreThanOnce() {
+    final var userId = generateId();
+    final var proxyUserId = generateId();
+
+    proxiesClient.createProxyRelationship(
+      ProxyRelationship.builder()
+        .userId(userId)
+        .proxyUserId(proxyUserId)
+        .build());
+
+    final var errors = proxiesClient.attemptToCreateProxyRelationship(
+        ProxyRelationship.builder()
+          .userId(userId)
+          .proxyUserId(proxyUserId)
+          .build())
+      .statusCode(is(422))
+      .extract().as(ValidationErrors.class);
+
+    assertThat(errors.getErrors().get(0).getMessage(),
+      is("Proxy relationship already exists"));
```

Comment:
A more rest-assured-like check:

```
      .statusCode(is(422))
      .body("errors[0].message", is("Proxy relationship already exists"));
```

Reply:
Please can you help me understand better what you mean by `a more rest assured like check`?

How is using the body method that uses a JSON path a better expression of this expectation?

To me, Rest Assured provides a broad set of tools for how to construct requests and check responses. I think the intention of that is to allow the consumers of the library to choose the means they prefer in any given situation.